ReplayKitExample/ReplayKitExample/ViewController.swift

```diff
@@ -254,39 +254,36 @@ class ViewController: UIViewController {
     }
 
     private func stopConference(error: Error?) {
-        // Stop recording the screen.
-        let recorder = RPScreenRecorder.shared()
-        recorder.stopCapture { (captureError) in
-            if let error = captureError {
-                print("Screen capture stop error: ", error as Any)
+        appScreenSource?.stopCapture { captureError in
+            if let captureError = captureError {
```

Comment:
The completion is fired on the main thread, correct?

Reply:
Correct always.